### ISSUE TITLE
Initializing with a custom class/id

### ISSUE THREAD

**Issue by rockers123** [fians/Waves#91]
Is it possible to initialize the plugin with a custom class? something like,

Waves.displayEffect('.my-custom-class');

Thanks

**Comment by hoxxep** [comment]
What would the class be attached to? If you can describe what you're trying to do with Waves that requires this extra class, then I'll understand what you mean - and why - a little better? :smile:

**Comment by rockers123** [comment]
Sorry, for the unclear question. 

I want to use waves effects on buttons in an existing project where I got buttons everywhere(around 100). Assigning the .wave-effect class to each and every buttons manually would cost me alot of time. It would make my life easier if I can directly attach the waves on <button> elements.

**Comment by fians** [comment]
I think it will be a nice idea. Actually, feature like that already added by @hoxxep ([see it here](https://github.com/fians/Waves/blob/master/src/js/waves.js#L312)), but it still really simple implementation and only accept DOM element as parameter. 

I will extend `Waves.attach()` to make it more flexible to use. But for now, you can call `Waves.displayEffect()` first then attach `waves-effect` class using javascript/jquery to your existing class element.

Example:

```
// vanilla javascript
var elements = document.getElementByClass('your-custom-class');
for (var a = 0; a < elements.length; a++) {
    elements[a].className += ' waves-effect';
}

// jQuery
$('.your-custom-class').addClass('waves-effect');
```

**Comment by rockers123** [comment]
Thank you very much for the awesome plugin and support. I think this will do the job

**Comment by hoxxep** [comment]
In pull request #108 I've upgraded the `Waves.attach(elements, classes)` function to allow you to use a string selector to apply the `.waves-effect` and any optional waves classes dynamically. Hopefully that helps!

**Comment by fians** [comment]
I just made a little changes to `Waves.attach()`. The function now will use array of classes string as parameter instead string of classes. It's just to improve code readability, just in case someone want to add more than 3 classes to Waves elements.

**Comment by fians** [comment]
Ok. It seems this feature is done. It will be shipped on v0.7.0
